src/main/java/com/itacademy/cms/service/UserService.java

```diff
@@ -8,11 +8,15 @@
 
   List<User> findAll();
 
-  User findById(Long id);
+//  User findById(Long id);
 
```

Comment:
Delete the commented code in this file if we don't need it

Reply:
Done